src/client.cpp

```diff
@@ -309,6 +312,12 @@ void SetSpeed(int speed)
   tvh->DemuxSpeed(speed);
 }
 
+void PauseStream(bool paused)
```

Comment:
using this function is wrong. you have to recalculate the conditions in read because player can run at lower speed, i.e. 0.8.